mycroft/configuration/mycroft.conf

```diff
@@ -118,7 +118,11 @@
     // priority skills to be loaded first
     "priority_skills": ["mycroft-pairing", "mycroft-volume"],
     // Time between updating skills in hours
-    "update_interval": 1.0
+    "update_interval": 1.0,
+    // Directory for aditional translations
+    "translations_dir": "~/.mycroft/translations",
```

Comment:
could this maybe be made relative to "data_dir" param?

Reply:
yes it could, but I would like to have translations in one logical place. 

having translations in one place makes it easier to acces, reviec and see which skills have additional translations. 

usecase is a auto-translation skill that autotranslate skills that are missing current language and a poodle sync skill that pulls down latest translations from translate.mycroft.ai (and maybe some day also push back translations)

Reply:
also it is easier to monitor one folder for changes and need of reload skills

Reply:
i'll leave this up to you and @forslund, no strong opinion either way

i don't see any issue re what you just said, it is still a single folder.... data_dir is a param in mycroft.conf, it is already used internally in a couple places, it would just use /opt/mycroft/translations

what i meant was
```python
translations_dir = join(data_dir, translations_path)
```

not data_dir from the skills or anything like that

Reply:
Oh I see - that were actual my first idea, but using ~/.mycroft makes sure there is read and write access across platforms and implementations.